containers/unit_tests/TestWithoutInitializing.hpp

```diff
@@ -24,6 +24,24 @@
 
 #include <../../core/unit_test/tools/include/ToolTestingUtilities.hpp>
 
+//! Get fence event message for @c ZeroMemset.
+template <typename execution_space>
+std::string fence_event_message_for_zeromemset(const execution_space& space) {
+  return std::string(space.name()).append(": ZeroMemset for");
+}
+
+//! @name Some tests are skipped for @c CudaUVM memory space.
+///@{
+#ifdef KOKKOS_ENABLE_CUDA
+#define GTEST_SKIP_IF_CUDAUVM_MEMORY_SPACE                            \
+  if constexpr (std::is_same_v<typename TEST_EXECSPACE::memory_space, \
+                               Kokkos::CudaUVMSpace>)                 \
```

Comment:
Thinking out lout: We recently deprecated the Kokkos_ENABLE_CUDA_UVM option so eventually that code path will not be reachable any more.  If we really care about guaranteeing we support it, we should probably instantiate based on memory spaces rather than relying on the default memory spaces associated to a specific execution space.

Reply:
@dalg24 I guess this is out of scope for this PR?